introduction.md

```diff
@@ -1,3 +1,11 @@
 Hi I'm AgroMonkey, I'm here to break some bricks.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?